helm/node-health-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Can we use a `Role` instead?

Reply:
Uncertain. Why? I copied this from `g8s-prometheus`.

Reply:
Can you check again please?